scripts/Packages/Package_Gullwing__QFP_SOIC_SO/ipc_gullwing_generator.py

```diff
@@ -102,7 +102,7 @@ def deviceDimensions(device_size_data):
             'lead_len': TolerancedSize.fromYaml(device_size_data, base_name='lead_len')
         }
         dimensions['has_EP'] = False
-        if 'EP_size_x' in device_size_data:
+        if 'EP_size_x_min' in device_size_data and 'EP_size_x_max' in device_size_data or 'EP_size_x' in device_size_data:
```

Comment:
This is ok for now but i think we will remove the old style sometime in the future.